src/screens/ExchangeModal.js

```diff
@@ -104,6 +106,10 @@ export default function ExchangeModal({
 
   const title = getInputHeaderTitle(type, defaultInputAsset);
   const showOutputField = getShowOutputField(type);
+  const priceOfEther = useEthUSDPrice();
+  const { genericAssets } = useSelector(({ data: { genericAssets } }) => ({
```

Comment:
can be `const genericAssets = useSelector(({ data: { genericAssets } }) => genericAssets);`